apps/io-session-manager-internal/src/controllers/unlock-user-session.ts

```diff
@@ -0,0 +1,56 @@
+import { httpAzureFunction } from "@pagopa/handler-kit-azure-func";
+import * as H from "@pagopa/handler-kit";
+
+import * as RTE from "fp-ts/ReaderTaskEither";
+import { pipe } from "fp-ts/lib/function";
+import { sequenceS } from "fp-ts/lib/Apply";
+
+import { FiscalCode, NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+
+import { SuccessResponse } from "../generated/definitions/internal/SuccessResponse";
+import { RequiredPathParamMiddleware } from "../utils/middlewares/required-path-param";
+import {
+  BlockedUsersService,
+  BlockedUsersServiceDeps,
+} from "../services/blocked-users-service";
+import { errorToHttpError } from "../utils/errors";
+
+type Dependencies = {
+  blockedUsersService: BlockedUsersService;
+} & BlockedUsersServiceDeps;
+
+export const makeUnlockUserSessionHandler: H.Handler<
+  H.HttpRequest,
+  | H.HttpResponse<SuccessResponse, 200>
+  | H.HttpResponse<unknown, 500>
```

Comment:
```suggestion
```

Reply:
removed in [69811ca](https://github.com/pagopa/io-auth-n-identity-domain/pull/296/commits/69811caa545b2a0e0e3567a479f0dc0ba6d4fd7a)